engine/include/cubos/engine/render/camera/camera.hpp

```diff
@@ -0,0 +1,27 @@
+/// @file
+/// @brief Component @ref cubos::engine::Camera
+/// @ingroup render-camera-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+#include <cubos/core/reflection/external/glm.hpp>
+#include <cubos/engine/api.hpp>
+#include <glm/mat4x4.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Generic component to hold the projection matrix of a specficic camera (either perspective or orthogonal)

```

Comment:
```suggestion
    /// @brief Generic component to hold the projection matrix of a specific camera (either perspective or orthogonal).

```

Minor spelling mistake :sunglasses: